client/src/components/Contact.style.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  align-items: center;
+  flex-flow: column wrap;
+  justify-content: center;
+`;
+
+export const H1 = styled.h1`
+  font-weight: bold;
```

Comment:
this font weight should be removed since you've declared a number below.
also, i think you need to add a margin to the top of your h1 so it doesn't go into your header!
![Screen Shot 2019-06-27 at 14 09 29](https://user-images.githubusercontent.com/37771591/60268845-4390b300-98e5-11e9-89cd-01418b6d6f0e.png)

![Screen Shot 2019-06-27 at 14 09 35](https://user-images.githubusercontent.com/37771591/60268838-3ffd2c00-98e5-11e9-8d21-afe04f6089cf.png)